README.md

```diff
@@ -11,24 +11,44 @@ This project is for OSIS which integrates [Scality RING](https://www.scality.com
 
 ## Running the Application
 
-### Command to run Docker Image
-A user needs to be logged into docker using URL: registry.scality.com.
-```sh
-    $ docker login -u <username in registry.scality.com> registry.scality.com
-```
-Once logged images can be pulled and run locally from either dev namespace using the hash or production namespace.
+### Commands to run Docker Image
+1. Docker images can be pulled locally from either dev namespace using the hash or production namespace.
+
+    Dev:
+    ```sh
+        $ docker pull registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
+    ```
+    Production image:
+    ```sh
+        $ docker pull registry.scality.com/vmware-ose-scality/vmware-ose-scality:<tag>
+    ```
+1. Generate a self-signed SSL certificate and store it in a binary PKCS#12 format file with extension `.p12` file. (Refer [here](#To-generate-PKCS12-file-for-self-signed-SSL-certificate))
+
+1. Create an environment variables file `application.properties`
+    - Sample `application.properties` file can be found [here](src/main/resources/application.properties).
+    - Update `server.ssl.key-store-password` and `server.ssl.key-alias` with key-store password and key-store alias of the `.p12` file respectively.
+    
+1. To run the docker image locally from either dev namespace using the hash or production namespace with the environment file `application.properties`
+
+    Dev:
```

Comment:
Since the only difference here is the path to the image, I would just assume the local build instead.

Reply:
Production image has a release tag instead of commit hash and release tags will be only made when release ready. Please correct me if I misunderstood the ask here.

Reply:
Yes, correct, but from an outside perspective trying this project for the first time, this line becomes very hard to use.

I can not copy-paste it, as it is missing a key piece of information I can not get from the readme alone. 
By using the local build, the readme is now complete without external information. Alternatively, if the `latest` tag was used such that the version was irrelevant that would work. But, we do not use the latest tag ( or any other generic terms like stable or prod ). 

My point is, without looking up extra information I can not use this step and part of that is to access eve, which is private or existing knowledge of the latest release tags that have been built.

Reply:
Sure I will add extra step for `docker build .` to build a locL image. Regarding release tags, anyone should be able to find the release tags on GIT and they can use that right. Correct me if my understanding of generating release tags is different

Reply:
FYI there is another PR in progress for release process

https://github.com/scality/vmware-ose-scality/pull/26

Reply:
In theory yes, a git tag would map to a docker image but that is purely by convention and without a docker hub tag list or similar seeing the "actual" tags is different. For example: say git tags include a prefix 'v' but docker images don't for whatever reason. My point being, it is just much easier to see the list of actual values over the assumed values from another source. Outside the scope of this issue tho so I will ignore it.